docs/src/targene/tmle.md

```diff
@@ -21,7 +21,9 @@ A set of predefined estimators is readily available and can be accessed by using
 - superlearning
 - tuned-xgboost
 
-all using TMLE as the meta statistical inference method. While these should cover most use cases, it may be useful to define a custom estimation strategy. This can be achieved by writing a small [Julia](https://julialang.org/) file described below.
+To see what the estimand files for these examples look like, please refer to the [TargetedEstimation.jl](https://github.com/TARGENE/TargetedEstimation.jl) package, located here: https://github.com/TARGENE/TargetedEstimation.jl/tree/main/estimators-configs.
```

Comment:
"estimator file". But maybe we should drop this section for now since I don't think this is even working atm...